ex5B/ex5.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Write a function that given a string, returns the first character that
+ * doesn't repeat in the whole string. Otherwise (all characters repeat),
+ * should return null.
+ *
+ * For example:
+ * firstUniqueChar("abcdedcba") === "e"
+ * firstUniqueChar("abcdeedcba") === null
+ * firstUniqueChar("abababcab") === "c"
+ * firstUniqueChar("aaaaaaaaa") === null
+ * firstUniqueChar("a") === "a"
+ */
+
+function firstUniqueChar(str) {
+
+	var doNotLookForCharList = '', searchChar = ''; 
+
+	for (var searchIndex = 0; searchIndex < str.length; searchIndex++) {
+
+		searchChar = str[searchIndex];
+
+		//  Let's see if we did look for searchChar
+		if (doNotLookForCharList.includes(searchChar) != true) {
```

Comment:
Try to comment out the `if (doNotLookForCharList.includes(searchChar) != true) {` and see if your code still works.